src/app/pages/MarginTradePage/hooks/useMargin_RecentTradesTable.ts

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { backendUrl, currentChainId } from 'utils/classifiers';
+import { RecentTradesDataEntry } from '../types';
+import { TradingPair } from 'utils/models/trading-pair';
+
+export const useMargin_RecentTradesTable = (pair: TradingPair) => {
+  const url = `${backendUrl[currentChainId]}/recentEvents/trade`;
+  const [data, setData] = useState<RecentTradesDataEntry[] | null>();
```

Comment:
You say the type here is either `RecentTradesDataEntry[]` or `null` but in fact, it's `undefined` :D . I don't see a reason why it should be `null` so do the following change.

Reply:
fixed